src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -676,9 +678,11 @@ public static function search(
             'digitised' => filter_var($digitised, FILTER_VALIDATE_BOOLEAN),
             'clean' => $clean,
             'precise' => filter_var($precise, FILTER_VALIDATE_BOOLEAN),
-            'limit' => (($page - 1) * 50) . ',50'
+            'itonesplaylistid' => $itonesplaylistid
         ];
-
+        if ($limit != null) {
```

Comment:
Erm, mainly because it's current only use was for show planner, which likes limits of results 👍